### ISSUE TITLE
Add nnapi serialization for module components in Wav2Vec2Model

### ISSUE THREAD

**Issue by davidavdav** [pytorch/pytorch#65058]
## 🚀 Feature
Running wav2vec2 model on accelerated mobile phone hardware, using nnapi

## Motivation

We've been able to run a quantized wav2vec2 model on a beefy mobile phone, but this takes a lot of CPU power, and is slow (because the model is large I suppose).  Being able to run on accelerated hardware via the nnapi would potentially speed up inference time. 

## Pitch

If the nnapi model serialization trick would work for a `torchaudio.Wav2Vec2Model`, then that would be really cool and big kudos for pytorch

## Alternatives

 - run slower
 - train a smaller model
 - run on GPU-backed cloud

## Additional context

I've tried monkey-patching `torch.backends._nnapi.serializer._NnapiSerializer.ADDER_MAP`, but after (guessing) the functionality of `aten::dim` as 
```python
def add_dim(self, node):
    assert node.inputsSize() == 1
    assert node.outputsSize() == 1

    _, in_oper = self.get_tensor_operand_by_jitval_fixed_size(node.inputsAt(0))
    res = len(in_oper.shape)
    output = node.outputsAt(0)
    self.add_constant_value(output, output.type(), res)
```
the next unsupported operation was `aten::conv1d`, and my guess it that that is too hard for me to do that, and potentially a whole lot of other operations, myself.

**Comment by axitkhurana** [comment]
@davidavdav we'd love to support this but I don't have any timeline for adding conv1d. Will update this issue when we get to it

**Comment by davidavdav** [comment]
Thanks, 

would model tracing, as an alternative,  be of help for us?  Or would that fail for the same reason (I realized later that the `aten::dim` would probably not have been accessed in the case od model tracing). 

—david
